hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -498,6 +499,9 @@ protected Bytes computeInternal(final MessageFrame frame) {
 		Bytes result;
 		ExpirableTxnRecord.Builder childRecord;
 		try {
+			if (frame.getRemainingGas().compareTo(gasRequirement) < 0) {
+				throw new InvalidTransactionException(INSUFFICIENT_GAS);
+			}
```

Comment:
Can we use `validateTrue(frame.getRemainingGas().compareTo(gasRequirement) >= 0, INSUFFICIENT_GAS)` to make the code cleaner?